configs/experiment_schema.json

```diff
@@ -0,0 +1,22 @@
+{
+  "title": "Experiment schema",
+  "type": "object",
+  "minProperties": 1,
+  "additionalProperties": false,
+  "properties": {
+    "rounds": {
+      "type": "array",
+      "minItems": 1,
+      "items": {
+        "url": "./announcement_schema.json#/definitions/prefixAnnouncement"
```

Comment:
An item in `rounds` is *not* a `prefixAnnouncement`, it's an object mapping a prefix or letter to a `prefixAnnouncement`. The call to `jsonschema.validate()` in `peering.py` succeeds because it doesn't seem to be doing anything. You can pass anything to it and it works:

```
>>> eg = peering.ExperimentController("x", "y")
>>> eg.schema
{'title': 'Experiment schema', 'type': 'object', 'minProperties': 1, 'additionalProperties': False, 'properties': {'rounds': {'type': 'array', 'minItems': 1, 'items': {'url': './announcement_schema.json#/definitions/prefixAnnouncement'}}, 'description': {'type': 'string'}, 'email': {'type': 'string', 'format': 'email'}}}
>>> exp
{'experiment': {'description': 'Testing prefix propagation of muxes amsterdam01, ufmg01 and seattle01', 'email': 'noc@peering.ee.columbia.edu', 'rounds': [{'wtf': 1}]}}
>>> jsonschema.validate(exp["experiment"], eg.schema)
>>>
```